services/booking-api/helpers/booking.js

```diff
@@ -0,0 +1,41 @@
+import * as request from '../../../libs/request';
+import params from '../../../libs/params';
+import config from '../../../config';
+
+const PATH = {
+  CREATE: 'create',
+  CANCEL: 'cancel',
+  GET: 'get',
+};
+
+const METHOD = {
+  POST: 'post',
+};
+
+function create(body) {
+  return sendBookingPostRequest(PATH.CREATE, body);
+}
+
+function cancel(body) {
+  return sendBookingPostRequest(PATH.CANCEL, body);
+}
+
+function get(body) {
+  return sendBookingPostRequest(PATH.GET, body);
+}
+
+async function sendBookingPostRequest(path, body) {
```

Comment:
The naming of this function implies that it will handle POST requests only. Can this be a problem in the future?

The 'get' function is also using this function. Is it correct to send post payload to the 'get' endpoint?
It's common to construct GET requests using path query parameters?

Sure I'm stupid not getting the picture :)